eq-terraform-deploy-image/Dockerfile

```diff
@@ -15,7 +15,8 @@ ENV PATH /google-cloud-sdk/bin:$PATH
 RUN apk add --no-cache python3 bash jq git curl && \
     git clone https://github.com/tfutils/tfenv.git ~/.tfenv && \
 	ln -s /root/.tfenv/bin/* /usr/local/bin && \
-	tfenv install 1.7.3
+	tfenv install 1.7.3 && \
```

Comment:
Question: why do we install the 1.7.3 version as well as the 1.9.3? is it for version support?

Reply:
Ah okay 👍